<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TravelMoment</class>
 <widget class="QWidget" name="TravelMoment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>旅行动态</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <!-- 发布区域 -->
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>发布新动态</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="contentEdit">
        <property name="placeholderText">
         <string>这一刻的想法...</string>
        </property>
        <property name="maximumHeight">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="imagePreview">
        <property name="text">
         <string>已选择 0 张图片</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectImageBtn">
        <property name="text">
         <string>选择图片（最多9张）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="publishBtn">
        <property name="text">
         <string>发布</string>
        </property>
        <property name="styleSheet">
         <string>background:#00aaee;color:white;font-weight:bold;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <!-- 动态流 -->
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="momentListLayout">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
